android/ndk_common.sh

```diff
@@ -7,14 +7,22 @@ export NDK_ARMV6_ROOT=$BUILD_DIR/android-ndk-r${NDK_ARMV6_VERSION}
 
 createNDKFolder()
 {
+    ndk_filename=android-ndk-r${NDK_VERSION}-linux-x86_64.zip
     rm -rf "$BUILD_DIR/android-ndk-r${NDK_VERSION}"
-    wget -c --no-verbose -O /tmp/ndk_${NDK_VERSION}.zip https://dl.google.com/android/repository/android-ndk-r${NDK_VERSION}-linux-x86_64.zip
-    unzip -qq /tmp/ndk_${NDK_VERSION}.zip -d $BUILD_DIR
+    echo Downloading $ndk_filename
+    wget -c --no-verbose -O /tmp/$ndk_filename https://dl.google.com/android/repository/android-ndk-r${NDK_VERSION}-linux-x86_64.zip
```

Comment:
Why you didn't change the url as well the way I suggested here https://github.com/BOINC/boinc/pull/4746#discussion_r873058417 ?

Reply:
I didn't understand that you want also the file name on the link. I will change that also.